pkg/services/sqlstore/migrations/signingkeys/migrations.go

```diff
@@ -0,0 +1,23 @@
+package signingkeys
+
+import "github.com/grafana/grafana/pkg/services/sqlstore/migrator"
+
+func AddMigration(mg *migrator.Migrator) {
+	var signingKeysV1 = migrator.Table{
+		Name: "signing_key",
+		Columns: []*migrator.Column{
+			{Name: "id", Type: migrator.DB_BigInt, IsPrimaryKey: true, IsAutoIncrement: true},
+			{Name: "key_id", Type: migrator.DB_NVarchar, Length: 255, Nullable: false},
+			{Name: "private_key", Type: migrator.DB_Text, Nullable: false},
+			{Name: "added_at", Type: migrator.DB_DateTime, Nullable: false},
```

Comment:
nit: `added_at`, is not this the same as `created_at`.

to be more consistent with column names

Reply:
Is it understandable it's only the time the key was added to the DB and not it's actual creation time?

Reply:
is there a creation time then?